src/ng-select/.eslintrc.json

```diff
@@ -0,0 +1,44 @@
+{
+  "extends": "../../.eslintrc.json",
+  "ignorePatterns": [
+    "!**/*"
+  ],
+  "overrides": [
+    {
+      "files": [
+        "*.ts"
+      ],
+      "parserOptions": {
+        "project": [
+          "src/ng-select/tsconfig.lib.json",
+          "src/ng-select/tsconfig.spec.json"
+        ],
+        "createDefaultProgram": true
+      },
+      "rules": {
+        "@angular-eslint/directive-selector": [
+          "error",
+          {
+            "type": "attribute",
+            "prefix": "lib",
+            "style": "camelCase"
+          }
+        ],
+        "@angular-eslint/component-selector": [
+          "error",
+          {
+            "type": "element",
+            "prefix": "lib",
+            "style": "kebab-case"
+          }
+        ]
+      }
```

Comment:
do you need to specify these since they are defined in [.eslintrc.json](https://github.com/ng-select/ng-select/pull/1996/files#diff-6884918dc8291219be508e05e28965b958c734def85324f3b53858ea4702090f) and it looks like this should extend that config? (The same applies to [src/ng-option-highlight/.eslintrc.json](https://github.com/ng-select/ng-select/pull/1996/files#diff-9aebdea382bb6ae71b2be933869895a6bc82eb729f67492cf024c90dc118d860))